guides/v3.2.0/object-model/computed-properties.md

```diff
@@ -20,8 +20,8 @@ Person = EmberObject.extend({
   lastName: null,
 
   fullName: computed('firstName', 'lastName', function() {
-    let firstName = this.get('firstName');
-    let lastName = this.get('lastName');
+    let firstName = this.firstName;
+    let lastName = this.lastName;
```

Comment:
I'd probably in-line these into the template string instead of assigning to local variables